docs/source/pages/Algorithm.rst

```diff
@@ -0,0 +1,36 @@
+Algorithm
+=========================================================
+
+The membrane curvature MDANAlysis module calculates mean and gaussian curvature 
+from surfaces derived by a group of reference. By default, the group of reference 
+implemented are lipid headgroups.
+
+The algorithm locates all the atoms in the group of reference and map each element 
+of the group to a grid of `m x n` dimensions. The dimensions of the grid are 
+determined by the length of the simulation box in the first frame of the 
+trajectory.
+
+The grid is then divided in `n x m` elements, with unit cells of width 2nm (default 
```

Comment:
```suggestion
The grid is then divided in `n x m` elements, with unit cells of width 2nm (default 
```

I'm not quite following, is `n` and `m` the same as in the previous line? Why is it now transposed? Could you specify that only the x and y coordinates are considered for this mapping?

Reply:
No, sorry, it's still `m x n`, not transposed. Change to specified coordinates added.